### PR TITLE
rename library name to avoid name conflicts from external

### PR DESCRIPTION
**Pull request by swimmingcarp**
1. rename to xxx.intel in Android.bp
2. support DRM_FORMAT_NV12 in hwc

Signed-off-by: yifang ma <yifangx.ma@intel.com>

### CODE REVIEW

File: platformminigbm.cpp [hunk 1]
```diff
@@ -83,20 +88,42 @@ int DrmMinigbmImporter::ImportBuffer(buffer_handle_t handle, hwc_drm_bo_t *bo) {
   bo->width = gr_handle->width;
   bo->height = gr_handle->height;
   bo->hal_format = gr_handle->droid_format;
-  bo->format = gr_handle->format;
+  if (need_modifier)
+    bo->format = DRM_FORMAT_NV12;
+  else
+    bo->format = gr_handle->format;
```
Comment (feijiang1) [inline platformminigbm.cpp]: the code need move down together with 
if (need_modifier == false) {

File: platformminigbm.cpp [hunk 2]
```diff
@@ -83,20 +88,42 @@ int DrmMinigbmImporter::ImportBuffer(buffer_handle_t handle, hwc_drm_bo_t *bo) {
...
+    bo->pitches[i] = gr_handle->strides[i];
+    bo->offsets[i] = gr_handle->offsets[i];
+    bo->gem_handles[i] = gem_handle;
   }
 
+  if (need_modifier == false) {
+    ret = drmModeAddFB2(drm_->fd(), bo->width, bo->height, bo->format,
+                        bo->gem_handles, bo->pitches, bo->offsets, &bo->fb_id, 0);
+  }
+  else {
```
Comment (feijiang1) [inline platformminigbm.cpp]: } else need place together

File: platformminigbm.cpp [hunk 3]
```diff
@@ -83,20 +88,42 @@ int DrmMinigbmImporter::ImportBuffer(buffer_handle_t handle, hwc_drm_bo_t *bo) {
...
+    uint32_t flag = DRM_MODE_FB_MODIFIERS;
+    uint64_t modifiers[HWC_DRM_BO_MAX_PLANES];
+    uint32_t numPlanes = gr_handle->base.numFds;
+
+    for (uint32_t i = 0; i < numPlanes; i++)
+      modifiers[i] = I915_FORMAT_MOD_Y_TILED;
```
Comment (feijiang1) [inline platformminigbm.cpp]: need add format check, if format is DRM_FORMAT_NV12_Y_TILED_INTEL, then modifiers set to I915_FORMAT_MOD_Y_TILED.
And we only support DRM_FORMAT_NV12_Y_TILED_INTEL, can add the assert

File: platformminigbm.cpp (hunk 1 above)
Comment (swimmingcarp) [reply platformminigbm.cpp]: Done.

File: platformminigbm.cpp (hunk 2 above)
Comment (swimmingcarp) [reply platformminigbm.cpp]: Done.

File: platformminigbm.cpp (hunk 3 above)
Comment (swimmingcarp) [reply platformminigbm.cpp]: Done.

**Pull request merged**